test/common/upstream/health_checker_impl_test.cc

```diff
@@ -3164,22 +3164,28 @@ TEST(HealthCheckEventLoggerImplTest, All) {
   NiceMock<MockClusterInfo> cluster;
   ON_CALL(*host, cluster()).WillByDefault(ReturnRef(cluster));
 
-  HealthCheckEventLoggerImpl event_logger(log_manager, "foo");
+  NiceMock<MockSystemTimeSource> system_time_source;
+  EXPECT_CALL(system_time_source, currentTime())
+      // This is rendered as "2009-02-13T23:31:31.234Z".
+      .WillRepeatedly(Return(SystemTime(std::chrono::milliseconds(1234567891234))));
+
+  HealthCheckEventLoggerImpl event_logger(log_manager, system_time_source, "foo");
 
   EXPECT_CALL(*file, write(absl::string_view{
                          "{\"health_checker_type\":\"HTTP\",\"host\":{\"socket_address\":{"
                          "\"protocol\":\"TCP\",\"address\":\"10.0.0.1\",\"resolver_name\":\"\","
                          "\"ipv4_compat\":false,\"port_value\":443}},\"cluster_name\":\"fake_"
-                         "cluster\",\"eject_unhealthy_event\":{\"failure_type\":\"ACTIVE\"}}\n"}));
+                         "cluster\",\"eject_unhealthy_event\":{\"failure_type\":\"ACTIVE\"},"
+                         "\"timestamp\":\"2009-02-13T23:31:31.234Z\"}\n"}));
```

Comment:
it's too bad there's not a reasonable way to have the timestamp written as the first field in the json, just to make it slightly more human readable